Doc/library/datetime.rst

```diff
@@ -2537,6 +2537,12 @@ Notes:
       string. Otherwise ``%Z`` is replaced by the returned value, which must
       be a string.
 
+      Note that ``strptime`` only accepts certain values for ``%Z``: ``UTC``
```

Comment:
I think this wording would be better (the extra "and" is throwing me off here):

```
  Note that ``strptime`` only accepts certain values for ``%Z``: any value
  in ``time.tzname`` for your machine's locale and the hard-coded values
  ``UTC`` and ``GMT``. So someone living in Japan may have ``JST``, ``UTC``,
  and ``GMT`` as valid values, but probably not ``EST``. 
```

Reply:
@pganssle no problem. Do you want me to rebase the pull request? with only one commit?

Reply:
did you also scrap `It will return a ``ValueError`` for any invalid strings.` on purpose?

Reply:
No, I just meant that there doesn't need to be any changes from "It will return a ``ValueError`` for nay invalid strings." on.

Reply:
OK I added it back. We are good to go.